src/extensions/Process.ts

```diff
@@ -40,4 +42,20 @@ export const Process = new RoAssociativeArray([
             },
         }),
     },
+    {
+        name: new BrsString("setManifest"),
```

Comment:
Hi @lkipke, @alimnios72 
Thank you for your feedback. I did try to use a similar approach but was unable to reach the needed result. Maybe it because of using the internal `cwd()` function. Then I found in the [codebase](https://github.com/sjbarag/brs/blob/35d45dfe276b06e137e681094076a3986bab175d/src/brsTypes/components/RoDeviceInfo.ts#L84) `set locale` function and decided to use that approach. 

I added changes to avoid using the additional function, please review 😊